embrace-android-features/src/main/kotlin/io/embrace/android/embracesdk/internal/capture/memory/MemoryWarningDataSource.kt

```diff
@@ -56,6 +56,7 @@ internal class MemoryWarningDataSource(
      * @param trimLevel the context of the trim, giving a hint of the amount of trimming.
      */
     override fun onTrimMemory(trimLevel: Int) {
```

Comment:
these don't work anymore if customers target SDK 34. This is something we should really deprecate for 7.0, if not entirely eliminated